Tools-Override/depProj.targets

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+***********************************************************************************************
+depProj.targets
```

Comment:
this the comment I mentioned above 

Note that depProj.targets is copied from Tools folder and then added FilterNugetPackages target. thanks to @ericstj who provided the changes there.

I don't think we need to do it in 2 steps as we can diff the version with the build tools one at any time